src/cascadia/Remoting/Monarch.cpp

```diff
@@ -201,6 +201,12 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
             _clearOldMruEntries(id);
         }
 
+        TraceLoggingWrite(g_hRemotingProvider,
+                          "Monarch_lookupPeasantIdForName",
+                          TraceLoggingWideString(winrt::hstring{ name }.c_str(), "name", "the name we're looking for"),

```

Comment:
`std::wstring` would be a better fit and allow SSO.

Reply:
```suggestion
                          TraceLoggingWideString(std::wstring{ name }.c_str(), "name", "the name we're looking for"),
```